doc/usage.md

```diff
@@ -168,7 +168,7 @@ Every `Error` object we pass back has the properties:
 For example, sending message to a non-existent user:
 
 ```js
-bot.sendMessage(nonExistentUserId, 'text').catch(error => {
```

Comment:
@GochoMugo I think, that my IDE (WebStorm) doesn't understand new syntax. I have just tried and yep, it's working despite errors... Then I pick ES6 it's suggest me to setup Babel Transpiler. So I can just ignore it and use ES6?